libs/expo/betterangels/src/lib/screens/AddEditClient/RelevantContacts/Contact/index.tsx

```diff
@@ -21,9 +29,25 @@ interface IContactProps {
 
 export default function Contact(props: IContactProps) {
   const { index, remove } = props;
-  const { control, setValue, watch } = useFormContext<
-    UpdateClientProfileInput | CreateClientProfileInput
-  >();
+  const {
+    control,
+    setValue,
+    watch,
+    formState: { errors },
+  } = useFormContext<UpdateClientProfileInput | CreateClientProfileInput>();
+
+  const contactPhoneNumber = watch(
+    `contacts[${index}].phoneNumber` as `contacts.${number}.phoneNumber`
+  );
+
+  useEffect(() => {
+    if (contactPhoneNumber === '') {
+      setValue(
+        `contacts[${index}].phoneNumber` as `contacts.${number}.phoneNumber`,
+        null
+      );
+    }
+  }, [contactPhoneNumber, index]);
```

Comment:
Same as here: https://github.com/BetterAngelsLA/monorepo/pull/549/files#r1783653802

Reply:
don't like this effect. lets walk through together